allennlp/tests/data/fields/sequence_label_field_test.py

```diff
@@ -65,25 +66,25 @@ def test_sequence_label_field_raises_on_incorrect_type(self):
         with pytest.raises(ConfigurationError):
             _ = SequenceLabelField([[], [], [], [], []], self.text)
 
-    def test_class_variables_for_namespace_warnings_work_correctly(self):
+    def test_class_variables_for_namespace_warnings_work_correctly(self, caplog):
```

Comment:
Nit: `caplog` is not a great variable name.

Reply:
This is actually an out-of-the-box pytest fixture so I unfortunately I think we're stuck with that name

Reply:
https://docs.pytest.org/en/latest/logging.html#caplog-fixture

Reply:
Oh, hmm, didn't realize that pytest did this magic. Ok 🤷‍♂️